ci/jenkins/templates/arm_jenkinsfile.groovy.j2

```diff
@@ -19,7 +19,7 @@
 
 {% call m.invoke_build(
   name='BUILD: arm',
-  node='ARM-SMALL',
+  node='ARM-GRAVITON3',
```

Comment:
Thanks, it would be useful to have a analysis of cost and the way we structure the tests. 

As of now running the UTs directly through e2e compilation can take up a lot of CI time. A lot of that comes from tests that are as a matter of fact integration tests

Would be great for us to isolate out a limited set of integration tests (in cases with `tests/arm_sve/`) and only run limited set of testcases over these would be useful. Like our `require_cuda` tag, while majority of tests do not have to go through the specific nodes